pkg/aws/scope/clients.go

```diff
@@ -31,6 +32,15 @@ func NewCloudfrontClient(session aws.Session, arn string, target runtime.Object)
 	return CloudfrontClient
 }
 
+// NewRoute53Client creates a new route53 API client for a given session
+func NewRoute53Client(session aws.Session, arn string, target runtime.Object) *route53.Route53 {
```

Comment:
I cannot see where this is called, but I guess this comes later right?

Reply:
Yes correct